papermill/execute.py

```diff
@@ -151,8 +151,8 @@ def execute_notebook(notebook,
         progress_bar (bool): Flag for whether or not to show the progress bar.
         log_output (bool): Flag for whether or not to write notebook output to stderr.
     """
-    print("Input Notebook:  %s" % get_pretty_path(notebook))
-    print("Output Notebook: %s" % get_pretty_path(output))
+    # print("Input Notebook:  %s" % get_pretty_path(notebook))
+    # print("Output Notebook: %s" % get_pretty_path(output))
```

Comment:
I think it is useful to keep printing the input and output paths. (If we decide it isn't useful we should delete the code instead of commenting it out)

Reply:
Get it, it doesn't matter much :)